digits/scheduler.py

```diff
@@ -214,6 +214,28 @@ def get_job(self, job_id):
             return None
         return self.jobs.get(job_id, None)
 
+    def get_related_jobs(self, job):
+        """
+        Look through self.jobs to try to find the Jobs
+        whose parent contains job
+        """
+        related_jobs = []
+
+        if 'Model' in job.job_type():
+            datajob = job.parent_job()[0]
```

Comment:
I think this is the only reference to `parent_job()` and it's only useful if the job is a `ModelJob` in which case the `dataset` attribute is returned. Wouldn't it be simpler to remove this `parent_job()` method altogether and do something like:
```
if isinstance(job, ModelJob):
    datajob = job.dataset
elif isinstance(job, DatasetJob):
    datajob = job
else:
    raise ValueError("Unhandled job type %s" % job.job_type())
```